analyses/molecular-subtyping-HGG/results/HGG_molecular_subtype.tsv

```diff
@@ -23,12 +23,19 @@ PT_279D9QZ8	7316-325	BS_1YTHM07J	BS_PKW05VZV	HGG, H3 wildtype	None documented	0.
 PT_2A5QN7QS	7316-10004	BS_GBXD55F1	BS_S6B9DVNS	HGG, H3 G35, TP53 loss	None documented	0.8102832451884409	1	0	0	0	p.R342*	NA	NA	NA	0	0	loss
 PT_2JDDX6TJ	7316-409	BS_B9QP40ER	BS_XZM79E42	DMG, H3 K28, TP53 activated	None documented	0.6442539090932794	1	0	0	0	p.R248Q	NA	NA	NA	1	1	activated
 PT_2QZJ4T95	7316-715	NA	BS_B1C6GZ84	HGG, To be classified	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
+PT_2WVW55DA	7316-1890	NA	BS_QAG1RDXN	HGG, To be classified	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
+PT_2WVW55DA	7316-1890	NA	BS_QAG1RDXN	IHG, ALK-altered	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
```

Comment:
This is a situation in which you will have to make sure to remove the HGG, To be classified - there should be unique `sample_id` in these files.

Reply:
There are still a few samples with duplicated sample_id, due to different Kids_First_Biospecimen_ID_DNA/RNA. Do we need to collapse there samples or we can leave them for now?